File: python/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    answer = []
    zero_sum = lottos.count(0)
    cnt = 0
    for lotto in lottos:
        if lotto in win_nums:
            cnt += 1
            
    max_value = min(6, 7-(cnt+zero_sum))
    min_value = min(6, 7-cnt)
    answer.append(max_value)
    answer.append(min_value)

    return answer

File: js/로또의 최고 순위와 최저 순위.js
function solution(lottos, win_nums) {
    var answer = [];
    
    let zero_count = 0
    let same_count = 0
    
    lottos.forEach(number => {
        if (number === 0) {
            zero_count++
        }
        else if (win_nums.includes(number)) {
            same_count++
        }
    })
    
    var max_value = Math.min(6, 7-(same_count+zero_count))
    var min_value = Math.min(6, 7-same_count)
    
    answer.push(max_value)
    answer.push(min_value)

    return answer;
}